config.go

```diff
@@ -61,18 +61,18 @@ func (e Env) String() string {
 	} else if e.Name != nil && e.Value != nil {
 		match := regKeyFromSecret.FindStringSubmatch(*e.Value)
 		if len(match) == 3 {
-			return fmt.Sprintf("Env \"%s\" with value set from key \"%s\" from Secret \"%s\"", *e.Name, match[2], match[1])
+			return fmt.Sprintf("Pair \"%s\" with value set from key \"%s\" from Secret \"%s\"", *e.Name, match[2], match[1])
```

Comment:
Nit, is `Pair` the right message from users point of view? It might be a little bit misleading imho, what about `Entry` or something similar?

Reply:
That's a reasonable point. Fixed.